### PR TITLE
Setup DB

### LINKED ISSUE #2: Setup DB

### PR DESCRIPTION
**Pull request by maclewan**
Closes #2

### CODE REVIEW

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt [hunk 1]
```diff
@@ -27,4 +33,38 @@ class LoginActivity : AppCompatActivity() {
         val myIntent = Intent(this, MainActivity::class.java)
         startActivityForResult(myIntent, 123)
     }
+
+    fun testDb(){
+        AsyncTask.execute {
+
+            //how to init db
+            try {
+                database = Room.databaseBuilder(
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt:42]: please extract this later to separate file

File: app/src/main/java/com/piwniczna/mojakancelaria/Models/ClientEntity.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "clients")
+data class ClientEntity(
+
+    @ColumnInfo(name = "Name")
+    var name: String ="",
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/Models/ClientEntity.kt:11]: I remember that we were talking about firstName and lastName, so are we staying with name only?

**Review by wiktoriasalamon — COMMENTED** [review]
Reviewed

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt (hunk 1 above)
Comment (wiktoriasalamon) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt:42]: You can create an issue directly from this comment

File: app/src/main/java/com/piwniczna/mojakancelaria/Models/ClientEntity.kt (hunk 2 above)
Comment (maclewan) [reply app/src/main/java/com/piwniczna/mojakancelaria/Models/ClientEntity.kt:11]: Tak i tak

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt (hunk 1 above)
Comment (maclewan) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt:42]: But there is already one issue about that so :P

Comment (wiktoriasalamon) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt:42]: this one - #6  ?

Comment (wiktoriasalamon) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt:42]: @sasuke5055

Comment (maclewan) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt:42]: Tak

Comment (wiktoriasalamon) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt:42]: Ok,  thanks :p

**Pull request merged**